src/EditorFeatures/Test/Utilities/BloomFilterTests.cs

```diff
@@ -106,6 +106,31 @@ public void TestEmpty()
             }
         }
 
+        [Fact]
+        public void TestCacheWhenEmpty()
+        {
+            _ = new BloomFilter(falsePositiveProbability: 0.0001, isCaseSensitive: false, []);
+
+            Assert.False(BloomFilter.BloomFilterHash.TryGetCachedEntry(out _, out _));
+        }
+
+        [Fact]
+        public void TestCacheAfterCalls()
```

Comment:
We need tests with vastly different Bloom filters, demonstrating we get expected probability results.

Reply:
Is this a general request, or is this something you think needs testing in light of the code now using a cache?